guides/source/getting_started.md

```diff
@@ -388,9 +388,9 @@ create and read. The form for doing this will look like this:
 ![The new article form](images/getting_started/new_article.png)
 
 It will look a little basic for now, but that's ok. We'll look at improving the
-styling for it afterwards.
+styling for it afterward.
 
-### Laying down the ground work
+### Laying down the groundwork
```

Comment:
Can you leave just this change, and the addition of `the` below, and revert other changes?
Others are correct as they are.